docker-compose.yml

```diff
@@ -179,11 +209,17 @@ services:
       - ./data/faf-java-api/logs:/logs
       - ./data/faf-java-api/repositories:/repositories
     expose:
+      - "8010" # HTTP API
       - "8011" # HTTP Management API
     ports:
-      - "8010:8010" # HTTP API
+      - "127.0.01:8010:8010" # For internal testing only
```

Comment:
O_o copy-paste kung-fu with missing dots? or there is some magic in that IP addresses? O_o

Reply:
Good catch.